admin/admin.go

```diff
@@ -6,83 +6,17 @@
 package admin
```

Comment:
Nit: Perhaps `admin_server.go` `admin.go` ? To follow the `metrics_server.go` convention ?